### ISSUE TITLE
Merge Conflicts

### ISSUE THREAD

**Issue by HL232** [HL232/bme590final#69]
Yo Stephen, both Matt and I are getting merge conflicts after you changed the database and img_processor code to include email stuff. What happened? 

(also why are we doing sendgrid email??? we totally don't need to)

We're gonna try to manually replace the files locally for now, but we can get to this tomorrow after your biochem final if you want.

**Comment by AznStevy** [comment]
what are you getting when you do a `git pull`?

**Comment by HL232** [comment]
I pulled from your develop-webserver branch earlier, and i didn't get any "pull merge conflicts." I can also push to my branch just fine. But now if I try to merge my branch onto fake_master I get a merge conflict. Same with Matt, he's trying to merge his own UIColor into UIEnhance but gets a merge error

**Comment by AznStevy** [comment]
i think that's because `fake_master` doesn't have updated code... what you could do is save all the files you changed, from your branch, do `git merge master` and then paste in your changed code to the files again. That's how i've been resolving merge conflicts.

**Comment by AznStevy** [comment]
@HL232 was it resolved with #70 or are there still issues on my side?

**Comment by HL232** [comment]
Yeah, I got it for myself. So I have access to fake_master locally, since I made that branch. So went onto fake_master, pulled fake_master, and then locally merged develop-webserver and my latest branch, pushed the fake_master again

LONG STORY SHORT, yeah, I got mine to fix. I'm on the phone with Matt, I'll see if I can walk him through it
